databaseutil/databaseutil.go

```diff
@@ -94,6 +94,59 @@ func GetPasswordForUserWithEmailAddress(emailAddress string) ([]byte, error) {
 	return password, nil
 }
 
+func StoreNewNote(authorId int64, content string, creationTime time.Time) (int64, error) {
```

Comment:
It looks like the convention in this file is to use the verb `insert` instead of `store`